src/Illuminate/Foundation/Console/OptimizeCommand.php

```diff
@@ -96,19 +100,22 @@ protected function compileClasses()
     }
 
     /**
-     * Get the node traverser used by the command.
+     * Get the class preloader used by the command.
      *
-     * @return \ClassPreloader\Parser\NodeTraverser
+     * @return \ClassPreloader\ClassPreloader
      */
-    protected function getTraverser()
```

Comment:
2.x requires us to build the traverser ourselves, 3.x provides a factory class that we use. So, if we'd have kept that method, we'd have only been calling it if the user had 2.x installed, which is breaking just the same as removing the method.